recipe/meta.yaml

```
@@ -1,24 +1,35 @@
 {% set name = "cuda-python" %}
-{% set version = "11.8.2" %}
+{% set version = "12.0.0" %}
 {% set major_version = version.split(".")[0]|int %}
 {% set minor_version = version.split(".")[1]|int %}
 {% set patch_version = version.split(".")[2]|int %}
+{% set target_name = "x86_64-linux" %}  # [linux64]
+{% set target_name = "ppc64le-linux" %}  # [ppc64le]
+{% set target_name = "sbsa-linux" %}  # [aarch64]
+{% set target_name = "x64" %}  # [win]
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://github.com/NVIDIA/cuda-python/archive/refs/tags/v{{ version }}.tar.gz
-  sha256: bfb36d314065b695decd23935ab008630cc956cbcd428d520106568fdeb7e644
+  sha256: 7df0f84c4b6210112353374a7d992bfd5bba9c97358b39ea03b61bd634ce9c8a
 
 build:
   number: 0
-  skip: true  # [py<37 or osx or cuda_compiler_version != "11.8"]
+  # TODO: Add `linux_aarch64`, `linux_ppc64le`, and `win_64` once supported on CUDA 12 inn conda-forge.
+  # xref: https://github.com/conda-forge/cuda-python-feedstock/issues/36
+  # xref: https://github.com/conda-forge/cuda-python-feedstock/issues/37
+  # xref: https://github.com/conda-forge/cuda-python-feedstock/issues/38
+  skip: true  # [aarch64 or ppc64le or win or osx or cuda_compiler_version != "12.0"]
   script:
     - {{ PYTHON }} -m pip install . --no-deps -vv
+  script_env:
+    # Ensure that CUDA includes can be found by the host compiler
+    - CUDA_HOME="${BUILD_PREFIX}/targets/{{ target_name }}"
   run_exports:
-    - cuda-python >=11.7.1,<12  # we need this because CUDA Python 11.7.1 has ABI breaks, see conda-forge/cuda-python-feedstock#15
+    - cuda-python >={{ major_version ~ ".0" }}
```

Can we put the upper bound back on this?

---

What about something like this?

```suggestion
    - {{ pin_compatible('cuda-python', min_pin='x', max_pin='x') }}
```